### PR TITLE
Fix-1434 Added dividers in Saving Account Transaction RecyclerView

### LINKED ISSUE #1434: No dividers in Savings Account Transaction RecyclerView
**Summary:**

There are no dividers in transaction recycler view 

**Steps to reproduce:**

Accounts -> Savings -> Transactions

**Expected behaviour:**

There should be dividers between the transactions.

**Observed behaviour:**

There are no dividers between the transactions.

**Device and Android version:**

Vivo V17
FunTouch OS_9.2

**Screenshots:**

I am attaching a collage of both the current and proposed views for it.

CollageMaker_20200318_172729703

### PR DESCRIPTION
**Pull request by rutvik-panchal**
Fixes #1434  

- Added dividers in Saving Account Transaction RecyclerView

**Screenshots:**

Screenshot_20200318_214803

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/layout/row_saving_account_transaction.xml [hunk 1]
```diff
@@ -35,15 +35,15 @@
             android:layout_height="wrap_content"
             android:layout_width="match_parent"
             android:text="@string/small_text"
-            android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium"
+            android:textAppearance="@style/Base.TextAppearance.AppCompat.Small"
```
Comment (ShivangiSingh17) [inline app/src/main/res/layout/row_saving_account_transaction.xml:38]: Why did you change it to small as it can work well with Medium also?

File: app/src/main/res/layout/row_saving_account_transaction.xml [hunk 2]
```diff
@@ -35,15 +35,15 @@
...
             android:textColor="@color/gray_dark"/>
 
         <TextView
             android:id="@+id/tv_transaction_detail_data"
             android:layout_height="wrap_content"
             android:layout_width="match_parent"
             android:text="@string/small_text"
-            android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium"
+            android:textAppearance="@style/Base.TextAppearance.AppCompat.Small"
```
Comment (ShivangiSingh17) [inline app/src/main/res/layout/row_saving_account_transaction.xml:46]: .

File: app/src/main/res/layout/row_saving_account_transaction.xml (hunk 1 above)
Comment (rutvik-panchal) [reply app/src/main/res/layout/row_saving_account_transaction.xml:38]: The text on xml represented it as small text and i found that it created inconsistency in size and it look irregular.
If that is wrong to do so, i will take it back.

**Comment by jawidMuhammadi** [comment]
This PR has conflicts and the view already has dividers, so closing this one.
